interfaces/mount/entry.go

```diff
@@ -141,3 +143,29 @@ func ParseEntry(s string) (Entry, error) {
 	e.CheckPassNumber = cpn
 	return e, nil
 }
+
+// LoadFSTab reads and parses an fstab-like file.
+//
+// The supported format is described by fstab(5).
+func LoadFSTab(reader io.Reader) ([]Entry, error) {
+	var entries []Entry
+	scanner := bufio.NewScanner(reader)
+	for scanner.Scan() {
+		s := strings.TrimSpace(scanner.Text())
+		if i := strings.Index(s, "#"); i != -1 {
+			s = strings.TrimSpace(s[0:i])
+		}
```

Comment:
to me at least it'd make more sense if you trimmed the comment first, and trimmed the space second. Like
```go
s := scanner.Text()
if i := strings.IndexByte(s, '#') {
    s = s[:i]
}
s = strings.TrimSpace(s)
```